### PR TITLE
Bioperl genbank2gff3

### PR DESCRIPTION
**Pull request by erasche**
Not all of the bioperl tools will be worth wrapping, especially when covered by other suites (e.g. seqtk), but some are definitely of interest. 

Done:

- genbank2gff3

Maybe todo? (Feel free to edit this list)

- bp_mutate
- bp_flanks
- ?

### CODE REVIEW

**Comment by bgruening** [comment]
@erasche do you want to support this over time?
I'm more inclined to bet on biopython and work further on this: https://github.com/bgruening/galaxytools/tree/master/tools/biopython/converters

**Comment by erasche** [comment]
@bgruening there's simply no biopython alternatives to this stuff. :/ So,
yes. I'm willing to support it because I desperately need it for chado.

Updates to your converters coming soon btw :-)

søn. 16. aug. 2015, 14.36 skrev Björn Grüning <notifications@github.com>:

**Comment by erasche** [comment]
@bgruening given your comments, I've:

- moved out of WIP
- removed anything else from the TODO list. I'll leave the rest of the bioperl suite alone, but I definitely need this tool.

:)

File: tools/bioperl/bp_genbank2gff3.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<tool id="bp_@EXECUTABLE@" name="BioPerl @EXECUTABLE@" version="1.0">
```
Comment (bgruening) [inline tools/bioperl/bp_genbank2gff3.xml]: I would prefer a more readable name. I don't think that our users care about the underlying software (BioPerl).
`GenBank to gff3 converter`?

File: tools/bioperl/bp_genbank2gff3.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    #if str($sofile.sofile) == "url":
+        "${sofile.so_url}"
+    #else:
+        live
+    #end if
+#end if
+--outdir -
+--ethresh $ethresh
+$model
+--typesource $typesource
```
Comment (bgruening) [inline tools/bioperl/bp_genbank2gff3.xml]: "${typesource}"

File: tools/bioperl/bp_genbank2gff3.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      <test>
+          <param name="genbank" value="seq.gb" />
+          <output name="gff3" file="seq.gb.1.gff" ftype="gff3" lines_diff="4"/>
+      </test>
+  </tests>
+  <help><![CDATA[
+**What it does**:
+
+Convert genbank to gff3
+      ]]></help>
```
Comment (bgruening) [inline tools/bioperl/bp_genbank2gff3.xml]: BioPerl ref?

File: tools/bioperl/bp_genbank2gff3.xml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+$genbank
+> $gff3]]></command>
+  <inputs>
+    <param label="Genbank file" name="genbank" type="data" format="gb"/>
+    <param name="noinfer" truevalue="" falsevalue="--noinfer" checked="true" type="boolean" label="Infer exon/mRNA subfeatures"/>
+    <conditional name="sofile" label="Sequence Ontology">
+        <param name="sofile" label="Sequence Ontology File" type="select">
+            <option value="__none__" selected="True">None specified</option>
+            <option value="live">Latest SO</option>
```
Comment (bgruening) [inline tools/bioperl/bp_genbank2gff3.xml]: s/SO/Sequence Ontolog/

**Comment by erasche** [comment]
Ping @bgruening

**Comment by bgruening** [comment]
And again a PR I missed. Sorry Eric!

**Comment by erasche** [comment]
no problem :) thanks!

**Pull request merged**